### ISSUE TITLE
Unable to customize status-right config

### ISSUE THREAD

**Issue by Clumsy-Coder** [egel/tmux-gruvbox#23]
I'm trying to use another plugin https://github.com/MunifTanjim/tmux-mode-indicator to display current tmux mode with the current `status-right`.

It seems tmux-gruvbox to be overriding the custom config.

Config:
```tmux
set -g status-right "#[bg=colour237,fg=colour239 nobold, nounderscore, noitalics]#[bg=colour239,fg=colour246] %Y-%m-%d  %H:%M #[bg=colour239,fg=colour248,nobold,noitalics,nounderscore]#[bg=colour248,fg=colour237] #h #{tmux_mode_indicator} "
```

Steps to reproduce
1. Load gruvbox plugin
2. Load `tmux-mode-indicator` plugin
3. Load custom `status-right` config
4. Custom status-right not updated

NOTE: it only works if 
1. Load the gruvbox plugin
5. Load custom `status-right` config
6. disable the gruvbox plugin from the tmux config (Tmux plugin manager)
7. reload tmux config `tmux source-file ~/.tmux.conf`

this is what I'm hoping to display
![image](https://user-images.githubusercontent.com/19594044/161166345-e9fa3c4f-a540-4313-b455-ba892f95c486.png)

**Comment by mvaldes14** [comment]
have the same situation, would love for the theme to allow us to pass some custom fields.

**Comment by realzhujunhao** [comment]
After installation, edit ~/.tmux/plugins/tmux-gruvbox/tmux-gruvbox-dark.conf  (or tmux-gruvbox-light.conf)
Remove the line starts with `set-option -g status-right`.
Then disable and enable gruvbox again.
This worked for me, but I guess it won't survive an update. :)

**Comment by egel** [comment]
Wow! First of all thanks for a lot of effort to document this.

**TLDR;** 
I am coming to you with good news. While adding new features, I hopefully fix the problem with interpreting the values from other plugins like [tmux-mode-indicator](https://github.com/MunifTanjim/tmux-mode-indicator) that make it hard to show up in the theme.
In the 

--- 

**A cup of details**

Currently, I am working on expanding the theme with some customization options (as many users reported) and being more flexible with what the users may want to define in each section. This issue hopefully should be resolved after merging #31.

If you want you may test it already in #31 (it may require a setup plugin for development -> see more of the new section in readme), but after a few tests I hope it may look like you may desire (see sample screenshot below).

![image](https://github.com/user-attachments/assets/1ba15f16-9844-4ad2-a2ab-9061ccb233d2)

**Comment by egel** [comment]
done, closing after merging release/v2.0.0
